pycode/memilio-epidata/memilio/epidata/cleanData.py

```diff
@@ -53,13 +53,18 @@ def clean_data(
         @param vaccination Defines if data generated by getVaccinationData.py should be deleted. Default = False
         @param commuter Defines if data generated by getCommuterData.py should be deleted. Default = False
         @param testing Defines if data generated by getTestingData.py should be deleted. Default = False
-        @param hdf5 Defines if files with ending json [Default] or hdf5 should be deleted.
+        @param Defines if files with ending json should be deleted. Default = False
+        @param hdf5 Defines if files with ending hdf5 should be deleted. Default = False
+        @param txt Defines if files with ending txt should be deletd. Default = False
         @param out_path Defines path where data should be deleted.
         """
-
-    ending = "json"
+    file_types = []
+    if json:
+        file_types += "json"
```

Comment:
I don't think that's going to work properly. Here we should add a list element, i.e. file_types += ["json"]

On the other hand, each addition initializes a new list, where the data is then copied in.
It would be better to use append here, i.e. file_types.append("json")

Reply:
Since `file_types` is already a list this works. I used append anyway.